go/vt/vtgate/planbuilder/operators/operator_test_data.txt

```diff
@@ -444,35 +441,3 @@ OuterJoin: {
 	}
 	Predicate: t.id = y.t_id
 }
-
-update tbl set col1 = apa
-Update {
-	table: 	TableSet{0}:tbl
-assignments:
-	col1 = apa
-}
-
-update tbl set col1 = 'apa', col2 = 1337 where id = 12 and name = 'gangal'
-Update {
-	table: 	TableSet{0}:tbl where id = 12 and `name` = 'gangal'
-assignments:
-	col1 = 'apa'
-	col2 = 1337
-}
-
-update user set u = 1 where id = (select id from user_extra where id = 42)
-SubQuery: {
-	SubQueries: [
-	{
-		Type: PulloutValue
-		Query: 	QueryGraph: {
-		Tables:
-			TableSet{1}:user_extra where id = 42
-		}
-	}]
-	Outer: 	Update {
-		table: 	TableSet{0}:`user` where id = (select id from user_extra where id = 42)
-	assignments:
-		u = 1
-	}
-}
```

Comment:
they need route planning to happen, and these tests do not have correct vschema to do that. I felt like we could lean on the plan tests instead of making this test more similar to the plan_tests

Reply:
What's missing in the `FakeSI` to do route planning?

Reply:
I tried adding it, but it requires a full vschema, and that seems like a distraction here. it would be nice to have more unit tests, but I don't think I feel they are worth the cost in this particular instance